cmd/txtdirect/go.mod

```diff
@@ -0,0 +1,23 @@
+module txtdirect
+
+go 1.12
+
+require (
+	github.com/SchumacherFM/mailout v1.2.0
+	github.com/captncraig/caddy-realip v0.0.0-20170918004412-5dd1f4047d0f
+	github.com/golang/freetype v0.0.0-20170609003504-e2365dfdc4a0 // indirect
+	github.com/gorilla/sessions v1.1.3 // indirect
+	github.com/juju/ratelimit v1.0.1 // indirect
+	github.com/lucas-clemente/quic-go v0.11.1 // indirect
+	github.com/mholt/caddy v1.0.0-beta2
+	github.com/miekg/caddy-prometheus v0.0.0-20190322143946-eb0f4d1615b0
+	github.com/prometheus/client_golang v0.9.2 // indirect
+	github.com/quasoft/memstore v0.0.0-20180925164028-84a050167438 // indirect
+	github.com/steambap/captcha v1.3.0 // indirect
+	github.com/txtdirect/txtdirect v0.2.0
```

Comment:
We are importing ourselves with an old version. This would also lead to issues locally as development changes wouldn't be picked up.